docs/pages/desktop-access/rbac.mdx

```diff
@@ -88,6 +96,18 @@ For desktops discovered via LDAP, Teleport applies the following labels automati
 | `teleport.dev/is_domain_controller` | `primaryGroupID`                                                                                | `true`                        |
 | `teleport.dev/ou`                   | Derived from `distinguishedName`                                                                | `OU=IT,DC=goteleport,DC=com`  |
 
+Additionally, users can configure LDAP attributes which will be converted into
+Teleport labels. For example, consider the following Desktop Access configuration:
+
+```yaml
+discovery:
+  label_attributes:
+    - location
+```
+
+For a desktop with a `location` attribute of `Oakland`, Teleport would apply a
```

Comment:
Is there somewhere a list of possible LDAP attributes that we can use? Maybe we could link to it, so users would know which one they can use?

Reply:
Added a link to the Microsoft attributes, but not all of these will be present on the computer object, and there may even be custom attributes.

Good idea overall though, thanks!